mobile/src/main/java/org/openhab/habdroid/background/BackgroundTasksManager.kt

```diff
@@ -699,6 +716,25 @@ class BackgroundTasksManager : BroadcastReceiver() {
                 }
                 ItemUpdateWorker.ValueWithInfo(mode)
             }
+            VALUE_GETTER_MAP[PrefKeys.SEND_BLUETOOTH_DEVICES] = { context, _ ->
+                fun isConnected(device: BluetoothDevice): Boolean {
+                    return try {
+                        val m: Method = device.javaClass.getMethod("isConnected")
+                        m.invoke(device) as Boolean
+                    } catch (e: Exception) {
+                        throw IllegalStateException(e)
+                    }
+                }
+
+                val bm = context.getSystemService(BLUETOOTH_SERVICE) as BluetoothManager
+
+                val devices = bm.adapter.bondedDevices
+                    .filter { isConnected(it) }
+                    .joinToString("|") { device -> device.address }
```

Comment:
Getting the alias is only supported on API 30+. Maybe add a second pref that sends the alias on 30+ and address on all versions?